Gemfile

```diff
@@ -28,6 +28,7 @@ gem "octokit"
 gem "omniauth-github"
 gem "omniauth-google-oauth2"
 gem "pagerduty", ">= 4.0"
+gem "pg-aws_rds_iam", github: "haines/pg-aws_rds_iam", ref: "fb91b9232837e350aa9c8440b7340346adae845e"
```

Comment:
It would be better if we could set up the Gemfile so that this is only used if needed. However, that applies to other optional features (e.g. pagerduty, omniauth), and not related to this PR, so we can do that later.

Reply:
I'd prefer for aws grouping to be a separate PR, could precede this one. I'm not clear on this freeze mechanism. Not clear whether you think this `pg-aws_rds_iam` should be part of a wider aws group, or optional on its own

Reply:
we're already only requiring `pg-aws_rds_iam` when enabled by config. I'm not sure how your production intends to setup `BUNDLER_WITHOUT`

Split out to a group `aws_rds_iam`, but it seems like what you're outlining you'd want this outside freeze in your production, but we'd want it part of freeze in our production